### PR TITLE
Add warning about g-suite accounts to upcoming documentation

### PR DESCRIPTION
**Pull request by pironic**
I paid my nest developer fee with my g-suite account even though i manage my nest devices with my gmail account, thinking i could invite the gmail account. This is unfortunately not the case. Just a small note might be helpful to others as well.

## Type of change

- [x] Spelling, grammar or other readability improvements (~`current`~`rc` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/home-assistant.io/pull/14855
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This PR needs to be based from and targeting the `next` branch.

Please change the target for this PR to `next` and git rebase this PR if needed.

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -37,12 +37,16 @@ You will need to follow the instructions in [Device Access Registration](https:/
 Quick Start Guide:
 
 - Accept the Terms of Service.
-- Pay a fee (currently US$5).
+- Pay a fee (currently US$5).
```
Comment (zsarnett) [inline source/_integrations/nest.markdown]:
```suggestion
- Pay a fee (currently US$5).
```

File: source/_integrations/nest.markdown [hunk 2]
```diff
@@ -37,12 +37,16 @@ You will need to follow the instructions in [Device Access Registration](https:/
...
 - Accept the Terms of Service.
-- Pay a fee (currently US$5).
+- Pay a fee (currently US$5). 
 - Register in the Device Access Console to get a `project_id`.
 - Authorize your Google Account and create OAuth credentials to get a `client_id` and `client_secret`.
 - Enable pubsub events in the Device Access Console (creates a topic).
 - Create a pull subscription to get a `subscription_id`.
 
+<div class='note warning'>
+It is not currently possible to share/be invited to a home with a G-Suite account. Make sure that you pay the fee with an account that has access to your devices.
```
Comment (zsarnett) [inline source/_integrations/nest.markdown]:
```suggestion
It is currently not possible to share/be invited to a home with a G-Suite account. Make sure that you pay the fee with an account that has access to your devices.
```

**Comment by pironic** [comment]
I hope i did that correctly...

**Comment by zsarnett** [comment]
image

uhh nope

**Comment by pironic** [comment]
> uhh nope

oh boy... that second attempt made things even worse. Am i able to close this pull request and just create a new one from the get go in the right place? rebase is new to me.

**Comment by zsarnett** [comment]
yes

**Pull request closed without merging**